src/Selector/SingleSelector.tsx

```diff
@@ -106,6 +108,12 @@ const SingleSelector: React.FC<SelectorProps> = (props) => {
         />
       </span>
 
+      <SelectNativeInput
```

Comment:
There exist a input for search usage. Seems that one should be aria hidden?

Reply:
The role of the search input is a `combobox`, which contains many `aria-*` attributes that should not be aria hidden.

`<SelectNativeInput />` is already aria hidden.

Reply:
Why make SelectNativeInput aria hidden?

Reply:
I've changed the code:

```diff
export default React.forwardRef<HTMLInputElement, SelectNativeInputProps>(
  function SelectNativeInput(props, ref) {
    const { prefixCls, className, ...rest } = props;

    return (
      <input
-       aria-hidden
-       tabIndex={-1}
        type="hidden"
-       readOnly
        ref={ref}
        className={classNames(`${prefixCls}-native-input`, className)}
        {...rest}
      />
    );
  },
);
```

Thank you. You're right. `aria-hidden` shouldn't be set. But `type="hidden"` was already set. The screen reader still can't read it. 

Should we remove the `type='hidden'` and add some accessibility attributes to make it accessible to screen readers?

Reply:
I guess could use `opacity: 0` + `0 width & height` to let it invisible on the screen but still workable with reader.

Reply:
done

```diff
import classNames from 'classnames';
import React from 'react';

+ const visuallyHidden: React.CSSProperties = {
+   position: 'absolute',
+   overflow: 'hidden',
+   width: 1,
+   height: 1,
+   border: 0,
+   margin: -1,
+   padding: 0,
+   clip: 'rect(0 0 0 0)',
+   whiteSpace: 'nowrap',
+ };

interface SelectNativeInputProps extends React.InputHTMLAttributes<HTMLInputElement> {
  prefixCls?: string;
}

export default React.forwardRef<HTMLInputElement, SelectNativeInputProps>(
  function SelectNativeInput(props, ref) {
    const { prefixCls, className, style, ...rest } = props;

    return (
      <input
-       type="hidden"
        ref={ref}
        className={classNames(`${prefixCls}-native-input`, className)}
+       style={{ ...visuallyHidden, ...style }}
        {...rest}
      />
    );
  },
);
```